service/src/main/java/bio/terra/cbas/dependencies/wes/CromwellClient.java

```diff
@@ -31,15 +33,30 @@ public CromwellClient(
     singletonHttpClient = new ApiClient().getHttpClient();
   }
 
-  private ApiClient getApiClient()
+  public ApiClient getWriteApiClient()
       throws DependencyNotAvailableException, AzureAccessTokenException {
+    String uri =
+        dependencyUrlLoader.loadDependencyUrl(DependencyUrlLoader.DependencyUrlType.CROMWELL_URL);
+
+    ApiClient apiClient = new ApiClient().setBasePath(uri);
+    apiClient.setHttpClient(singletonHttpClient);
+    apiClient.addDefaultHeader(
+        "Authorization",
+        "Bearer " + credentialLoader.getCredential(CredentialLoader.CredentialType.AZURE_TOKEN));
```

Comment:
Just to clarify my understanding, this will in later ticket(s) be changed to using the token passed in the user's request right? 🤔

Reply:
Yes, that's correct